components/reddit/reddit.app.js

```diff
@@ -213,7 +213,7 @@ module.exports = {
         })
       );
     },
-    async searchSubreddits(after, query = "") {
+    async searchSubreddits(after, query = "") {      
```

Comment:
Same

Reply:
This is fixed as per the VSCode settings change requested elsewhere.